website/docs/components/flyout/partials/guidelines/guidelines.md

```diff
@@ -13,40 +80,56 @@ The Flyout header features several properties to better communicate the purpose
 
 ### Title icon
 
-- An icon paired with the title can help reinforce the purpose and function of the Flyout while also drawing the eye to the header and title area.
-- The communication of the purpose of the Flyout should not rely solely on an icon, instead the title should be explicit and pragmatic while the icon provides visual support.
+**With title icon**
```

Comment:
@didoo Would it be better for now to use images for these examples?

Reply:
Probably is not the "right" solution, because I see the same problem in `Modal` and I would't be surprised to find it on other components too. I think we have to live with that for now, but think of a more systemic solution to the problem of _how do we mix content and component examples and make it visually clear where a component start/ends in the flow of the page, in those cases where the component/subcomponent doesn't have a border that delimits its bounding box._